client/src/components/VideoCards.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faThumbsUp, faThumbsDown } from "@fortawesome/free-solid-svg-icons";
+
+function VideoCard({ title, url, rating, onRemove }) {
+  const videoId = url.split("v=")[1];
```

Comment:
Is it possible a URL didn't contain `v=`? What would happen here if this happened? Is there a way that could be handled more smoothly?